codeworld-base/src/Internal/CodeWorld.hs

```diff
@@ -22,7 +22,9 @@ module Internal.CodeWorld
     , animationOf
     , activityOf
     , groupActivityOf
+    , CW.unsafeGroupActivityOf
```

Comment:
This shouldn't be exported.  There's no need for the unsafe version to be exported from `codeworld-base`, because the monad bind operator isn't exported.  I'm not sure what you were trying to do, but there should be a different way.